roc/receiver.go

```diff
@@ -34,6 +39,38 @@ func OpenReceiver(rocContext *Context, receiverConfig *ReceiverConfig) (*Receive
 	return (*Receiver)(receiver), nil
 }
 
+func (r *Receiver) Bind(portType PortType, proto Protocol, a *Address) error {
+	errCode := C.roc_receiver_bind(
+		(*C.roc_receiver)(r),
+		(C.roc_port_type)(portType),
+		(C.roc_protocol)(proto),
+		a.raw)
+	if errCode == 0 {
+		return nil
+	}
+	if errCode < 0 {
+		return ErrInvalidArgs
+	}
+	panic(fmt.Sprintf(
+		"unexpected return code %d from roc_receiver_bind()", errCode))
+}
+
+func (r *Receiver) ReadFloats(frame []float32) error {
+	if frame == nil {
+		return ErrInvalidArgs
+	}
+	errCode := C.rocGoReceiverReadFloats((*C.roc_receiver)(r), (*C.float)(&frame[0]), (C.ulong)(len(frame)))
```

Comment:
`frame[0]` will panic if frame is not nil, but its len is zero. So we should also check `if len(frame) == 0`. I think in this case we can just do nothing return nil (no error). Same for WriteFloats.